client/entry.tsx

```diff
@@ -4,15 +4,24 @@ import {
   createBrowserRouter,
   RouterProvider,
 } from 'react-router-dom';
+import cookies from 'js-cookie';
 
+import { Manifest } from 'app/types';
 import App from 'app/App';
 import dataRoutes from 'app/dataRoutes';
 
 const router = createBrowserRouter(dataRoutes);
+const manifestCookie = cookies.get('manifest');
+
+if (!manifestCookie) {
+  throw new Error('[FATAL]: Manifest missing');
```

Comment:
We should redirect to a 500 page when this occurs.

Reply:
Agreed - but that may need to wait until a proper error strategy is put into place. May or may not be in scope for this boilerplate.